src/middleware/requires-auth-middleware.ts

```diff
@@ -18,3 +28,22 @@ export function requiresAuthMiddleware(
     next();
   }
 }
+
+async function startGet(req: Request, res: Response): Promise<void> {
```

Comment:
I think `startGet` is a confusing name for this function here, could we call it something like `redirectToLogIn` instead?

Reply:
Ah yes sure